src/modules/dashboard/components/Suggestions/Suggestions.tsx

```diff
@@ -15,7 +15,7 @@ const displayName = 'dashboard.Suggestions';
 const Suggestions = ({ colonyAddress, domainId }: Props) => (
   <div>
     <SuggestionCreate colonyAddress={colonyAddress} domainId={domainId} />
```

Comment:
Not related to this PR, but I've noticed it, so I'll mentioned it.

There's no way to un-focus the "Suggest features" input field without causing a validation error.

![demo-cannot-unfocus-suggestions](https://user-images.githubusercontent.com/1193222/72727812-86252000-3b94-11ea-848e-1b77d3ffcb7e.gif)

The only way to do it is to change tabs, but otherwise every place you click, the field will scream at you that it's not valid anymore.

I know this is a polish thing and not something major, but I think this can be solved via yup's validation. 

Granted, at some later point.

Reply:
While on the "Create suggestion" subject, you can submit the suggestion title by pressing "Return", but you can't close the confirmation modal by pressing "Return".

So to create a new suggestion, you can't double-tap "return", you can only tap "return" then mouse-click "ok", which is a bit of an UX annoyance.

![Screenshot from 2020-01-20 14-50-08](https://user-images.githubusercontent.com/1193222/72727993-fc298700-3b94-11ea-961e-ead5c306aeb8.png)

Again, this is very much a polishing stage thing, and no where related to this PR, but I'm thinking you can open a issue for it when you're do so we remember to take care of it.

Reply:
> There's no way to un-focus the "Suggest features" input field without causing a validation error.

Yes, this is driving me nuts 🙂

Perhaps we can blur the input before resetting the form? I'll look into this.

> So to create a new suggestion, you can't double-tap "return", you can only tap "return" then mouse-click "ok", which is a bit of an UX annoyance.

This is a really good point. I'll open an issue for this, as I think this could be a nice improvement everywhere it's used.

Reply:
Update: I've fixed both of these:

1) Blur the input on success **before** resetting the form
2) When the confirm dialog opens, `OK` will be autofocused for easy (and obvious) submission